examples/platform/linux/BUILD.gn

```diff
@@ -43,6 +43,7 @@ source_set("app-main") {
     "ControllerShellCommands.h",
     "LinuxCommissionableDataProvider.cpp",
     "LinuxCommissionableDataProvider.h",
+    "NamedPipeCommands.cpp",
```

Comment:
Does this not work on Darwin?  It looked to me like it did....

Reply:
Yes, this works on both Darwin and Linux from my testing

Reply:
@yufengwangca How can the post-PR state work on Darwin if the code is only in platform/linux?

Reply:
It seems linux folder is for both Darwin and Linux, the build.gn file under it also contain the build flag for darwin